data/examples/declaration/value/function/arrow/proc-forms2.hs

```diff
@@ -4,16 +4,16 @@ bar0 f g h x =
   proc (y, z) ->
     (|   test (h f.(h g)  -<  (y x).y z)((h g)  .  h f-<y z  .  (y x))   |)
 
-bar1 f g x y = proc _ -> f -< x&&&g -< y
+bar1 f g x y = proc _ -> (f -< x)&&&(g -< y)
```

Comment:
What is the reason for adding parentheses?

Reply:
Otherwise it is not a valid syntax. Previously it used to work because that error was thrown after the parser; now the parsing fails.